.releaserc.js

```diff
@@ -46,7 +46,12 @@ module.exports = {
     ],
     '@semantic-release/git',
     join(__dirname, 'ci/yarnPublish.js'),
-    '@semantic-release/github',
+    [
+      '@semantic-release/github',
+      {
+        assets: 'release/*.tar.gz',
+      },
```

Comment:
This _should_ attempt to release any `.tar.gz` file under the `release` directory for any given package. I don't imagine it'll fail if `release` doesn't exist (or is empty), but I'm unsure.